preprocess_observation_smogon.py

```diff
@@ -27,7 +27,7 @@ def preprocess_observation(I):
     global retval3
     # Force switch flag
     fs = False
-    I = I.split('\n')
+    I = I.split('\r\n')   # Pesky Windows line endings
```

Comment:
Must be some way to get platform-dependent line endings from sys or os packages. Stack overflow?

Reply:
There is